plugins/taper/run.go

```diff
@@ -134,8 +134,7 @@ func (a *Agent) startReadOutputItems(ctx context.Context, outputItems chan *api.
 					log.Errorf("Failed to create telemetry: %v", err)
 					return
 				}
-				// TODO format host to host.namespace (or whatever format we will decide on for hosts)
-				if !a.shouldTrace(telemetry.Request.Host) {
+				if !a.shouldTrace(telemetry.Request.Host, item.ConnectionInfo.ServerPort) {
 					log.Infof("Ignoring host: %v", telemetry.Request.Host)
```

Comment:
Same question, do we really want to log in Info level every traces that are ignored ?

Reply:
I think that if you not running in Info level (which you dont by default), It is a NOOP. If you think it is a crucial thing, we can think of not printing here, or change it to TRACE